<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>screenUploaderWindow</class>
 <widget class="QWidget" name="screenUploaderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_status_lbl">
     <property name="text">
      <string>status:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_url_lbl">
     <property name="text">
      <string>URL:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="txt_url">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>ftp://example.com</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_username_lbl">
     <property name="text">
      <string>username:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="spb_ticks">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="txt_username">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btn_startStop">
     <property name="text">
      <string>start</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lbl_password_lbl">
     <property name="text">
      <string>password:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="txt_password">
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lbl_save_lbl">
     <property name="text">
      <string>save every</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="lbl_ticks">
     <property name="text">
      <string>ticks</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLabel" name="lbl_info">
     <property name="text">
      <string>Specify upload parameter!</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lbl_status">
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
     <property name="text">
      <string>pausing</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
